### PR TITLE
Update the doc for tuning strategy

### PR DESCRIPTION
**Pull request by yiliu30**
## Type of Change

documentation 
## Description

detail description 
JIRA ticket: [ILITV-2743](https://jira.devtools.intel.com/browse/ILITV-2743)

## How has this PR been tested?

Pre-CI

## Dependency Change?

None

### CODE REVIEW

File: docs/source/tuning_strategies.md [hunk 1]
```diff
@@ -85,21 +85,30 @@ accuracy_criterion = AccuracyCriterion(
...
+
+Intel® Neural Compressor allows users to choose different tuning processes by specifying the quantization level (`quant_level`). Currently, the recognized `quant_level`s are `0`, `1`, and `"auto"`. For `quant_level` is `1`, the tuning process can be finer-grained controlled by setting the `strategy` field.
+
+- `0`: "Conservative" tuning. `0` starts with an `fp32` model and tries to quantize OPs into lower precision by **op-type-wise**. `0` can be useful to give users insights about the accuracy degradation after quantizing some OPs.
+
+- `1`: "Aggressive" tuning. `1` starts with the default quantization configuration and selects different quantization parameters. `1` can be used to achieve the performance. 
+
+- `"auto"` (default) Auto tuning. `"auto"` combines the advantages of `quant_level=0` and `quant_level=1`. Currently, it tries default quantization configuration, `0`, and [`basic`](./tuning_strategies.md#basic-tuning) strategy sequentially.
```
Comment (Kaihui-intel) [inline docs/source/tuning_strategies.md:97]: Invalid link: basic.

File: docs/source/tuning_strategies.md [hunk 2]
```diff
@@ -114,38 +123,72 @@ conf = PostTrainingQuantConfig(
...
+
+- **Stage II**. Apply all recipes
 
-- **Stage I**. OP Type Wise Tuning
+    In this stage, it tries to apply all recipes.
 
-    In this stage, it tries to quantize the OPs as many as possible and traverse all OP type wise tuning configs. Note that,  the OP is initialized with different quantization modes according to the quantization approach.
+- **Stage III**. OP-Type-Wise Tuning
+
+    In this stage, it tries to quantize OPs as many as possible and traverse all OP type wise tuning configs. Note that,  the OP is initialized with different quantization modes according to the quantization approach.
```
Comment (Kaihui-intel) [inline docs/source/tuning_strategies.md:138]: "Note that,  the" should be "Note that, the".

File: docs/source/tuning_strategies.md [hunk 3]
```diff
@@ -114,38 +145,52 @@ conf = PostTrainingQuantConfig(
...
 
-- **Stage III**. Fallback Multiple OPs Accumulated
+- **Stage V**. Fallback OP One by One
 
-    In the final stage, it first sorted the OPs list according to the impact score in stage II, and tries to incrementally fallback multiple OPs to high precision according to the sorted OP list.
+    At this stage, it performs high-precision OP (FP32, BF16 ...) fallbacks one by one based on the tuning config with the best result in the previous stage, and records the impact of each OP. 
+
+- **Stage VI**. Fallback Multiple OPs Accumulated
+
+    At the final stage, it first sorted the OPs list according to the impact score in stage II, and tries to incrementally fallback multiple OPs to high precision according to the sorted OP list.
```
Comment (lvliang-intel) [inline docs/source/tuning_strategies.md:178]: the impact score in Stage II? 
Based on the above description, it looks like Stage V.

File: docs/source/tuning_strategies.md [hunk 4]
```diff
@@ -114,38 +145,52 @@ conf = PostTrainingQuantConfig(
...
 
 ### Usage
 
-`Basic` is the default strategy. It can be used by default with nothing changed in the `strategy` field of `TuningCriterion`. Classical settings are shown below:
+`Basic` is the default strategy for `quant_level`=`1`, It can be used by default with nothing changed in the `strategy` field of `TuningCriterion` after set the `quant_level`=`1` in `PostTrainingQuantConfig`. Classical settings are shown below:
```
Comment (lvliang-intel) [inline docs/source/tuning_strategies.md:182]: for `quant_level`=`1`, It  ===> it

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**